spec/typinggame_server/interactors/players/fetch_player_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'spec_helper'
+
+describe Interactors::Players::FetchPlayer do
+  let(:repository) { PlayerRepository.new }
+  let(:interactor) { described_class.new(repository: repository) }
+  let(:params) { 1 }
+
+  before do
+    repository.clear
+
+    repository.create(id: '1')
+    repository.create(id: '2')
+  end
+
+  context 'good input' do
```

Comment:
Let's use a more description context block.

ie. `When the player exists`

Reply:
@jgayfer fixed.